### PR TITLE
Fix typing with whitespaces

### PR DESCRIPTION
**Pull request by crazyk2**

### CODE REVIEW

File: test/unit/adb-commands-specs.js [hunk 1]
```diff
@@ -202,8 +202,8 @@ describe('adb commands', () => {
     }));
     describe('inputText', withMocks({adb}, (mocks) => {
       it('should call shell with correct args', async () => {
-        let text = 'some text';
-        let expectedText = 'some%stext';
+        let text = 'some text with spaces';
+        let expectedText = 'some%stext%swith%sspaces';
```
Comment (mykola-mokhnach) [inline test/unit/adb-commands-specs.js]: 👍

**Comment by imurchie** [comment]
@crazyk2 Thanks for submitting this! We'd love to get it in but cannot without the CLA being signed (https://cla.js.foundation/appium/appium-adb?pullRequest=217). Is this possible?

**Comment by crazyk2** [comment]
Of course! I signed the CLA.

**Comment by imurchie** [comment]
Hmm. Seems like your github username isn't associated with the commit.

screenshot 2017-04-21 13 34 42

https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user

**Pull request closed without merging**